source/manual/schemas.html.md

```diff
@@ -0,0 +1,42 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Schemas
+parent: "/manual.html"
+layout: manual_layout
+section: Tools
+last_reviewed_on: 2018-09-10
+review_in: 12 months
+---
+
+# Schemas used on GOV.UK
```

Comment:
You actually don't need this heading, it'll come from the `title` setting in the block above.

Reply:
👍 changes made